tests/output.rs

```diff
@@ -105,14 +211,19 @@ fn test_query_inline() {
 }
 
 #[test]
-fn test_vimgrep_mode() {
+fn test_specify_multiple_files() {
     assert_sorted_output(
         "rust_project",
         r#"
-            $ tree-sitter-grep --query-source '(function_item) @function_item' --language rust --vimgrep
-            src/helpers.rs:1:1:pub fn helper() {}
-            src/lib.rs:3:1:pub fn add(left: usize, right: usize) -> usize {
-            src/lib.rs:12:5:    fn it_works() {
-       "#,
+            $ tree-sitter-grep --query-source '(function_item) @function_item' --language rust src/lib.rs ./src/helpers.rs
+            ./src/helpers.rs:1:pub fn helper() {}
+            src/lib.rs:3:pub fn add(left: usize, right: usize) -> usize {
+            src/lib.rs:4:    left + right
+            src/lib.rs:5:}
+            src/lib.rs:12:    fn it_works() {
+            src/lib.rs:13:        let result = add(2, 2);
+            src/lib.rs:14:        assert_eq!(result, 4);
+            src/lib.rs:15:    }
+        "#,
```

Comment:
Refreshing my memory, if I passed in `src/*.rs` to tree-sitter-grep, that'll actually get expanded by my shell, right? So TSG will actually be passed a list of `.rs` files?

Reply:
Yes I was thinking about that too, I believe that is accurate. I tried adding one test with such a glob but something was failing (though the equivalent "actual" command-line was working as expected), maybe eg `shlex` got confused?

It looks like `ripgrep` also supports passing it "globs" - I think how this works (eg you can do this with ESLint or Prettier I remember) is that if you put it in single quotes eg `'**/*.rs'` then the shell doesn't expand it for you

But so that presumably is not currently supported. Eventually it would probably make sense to basically go through all of the eg `ripgrep` command-line options and see which ones also seem like they'd be appropriate/relevant and mimic/support them?